consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -674,3 +675,233 @@ def test_block_publisher_success_case(
         self.assertTrue(
             block_publisher.initialize_block(
                 block_header=mock_block.header))
+
+    @patch('sawtooth_poet.poet_consensus.wait_timer.WaitTimer')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.'
+                'SignupInfo')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.'
+                'PoetKeyStateStore')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.'
+                'ConsensusStateStore')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.factory')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.'
+                'ConsensusState')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.'
+                'ValidatorRegistryView')
+    @mock.patch('sawtooth_poet.poet_consensus.poet_block_publisher.utils')
+    def test_block_publisher_claim_readiness(
+            self,
+            mock_utils,
+            mock_validator_registry_view,
+            mock_consensus_state,
+            mock_poet_enclave_factory,
+            mock_consensus_state_store,
+            mock_poet_key_state_store,
+            mock_signup_info,
+            mock_poet_config_view,
+            mock_block_wrapper,
+            mock_wait_time):
```

Comment:
`mock_wait_time` should be after `mock_poet_config_view` as patches are done in reverse order of defining them and the patches at the function level come before the ones at class level.